library/src/main/java/com/android/volley/RequestQueue.java

```diff
@@ -203,6 +206,14 @@ public VolleyTimingLogger getTimingLogger() {
         return mTimingLogger;
     }
 
+    public void setRequestLogger(RequestLogger logger) {
```

Comment:
I added a todo. Will refactor separately. Since it requires changing all the constructors.